backend/tests/api/test_deliveries_api.py

```
@@ -1572,3 +1574,244 @@ def test_mark_loop_item_read_repairs_legacy_metadata_without_read_revisions(
+
+
+def test_rest_delivery_finalize_registers_wait_node_reference(
+    test_client: TestClient,
+    test_token: str,
+    test_db: Session,
+    test_user: User,
+    delivery_storage: FakeDeliveryStorage,
+) -> None:
+    """Every finalize entry point registers the wait-node reference binding."""
+
+    item, project, delivery_id, headers = _wait_reference_delivery_setup(
```

Ruff (0.16.1)

[warning] Unpacked variable `project` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)